packages/suite/src/support/messages.ts

```diff
@@ -1595,6 +1595,19 @@ const definedMessages = defineMessages({
         defaultMessage: 'Wipe device',
         id: 'TR_DEVICE_SETTINGS_BUTTON_WIPE_DEVICE',
     },
+    TR_DEVICE_SETTINGS_CUSTOM_FIRMWARE_BUTTON: {
+        defaultMessage: 'Install firmware',
+        id: 'TR_DEVICE_SETTINGS_CUSTOM_FIRMWARE_BUTTON',
+    },
+    TR_DEVICE_SETTINGS_CUSTOM_FIRMWARE_TITLE: {
+        defaultMessage: 'Install custom firmware',
+        id: 'TR_DEVICE_SETTINGS_CUSTOM_FIRMWARE_TITLE',
+    },
+    TR_DEVICE_SETTINGS_CUSTOM_FIRMWARE_DESCRIPTION: {
+        defaultMessage:
+            'It is possible to install custom firmware for your Trezor device if needed. Installing custom firmware will erase its memory and may make it unusable. Never use this process unless you really know what you are doing.',
```

Comment:
**may make it unusable**

Therefore I would at least add somewhere a link to [wiki](https://wiki.trezor.io/Firmware_downgrade).
Because if user downgrades `<=1.10.0` to `>1.10.0`, it breaks the device.

Reply:
I agree that since there is a button for it in the menu, there is a risk of misusing. But it was assumed from the beginning that this is the 'power-user' feature, so they have to know what they're doing.

Reply:
> so they have to know what they're doing

Without the right materials, you cannot know that you cannot downgrade to `1.9.1` from `1.10.1`

Reply:
Added in [`8c27c7dd`](https://github.com/trezor/trezor-suite/commit/8c27c7dd953acdbff7b3ef136e41f1bb160dd307).